{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"nameInUseException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/NameInUseException","preciseIdentifier":"s:19AWSAlexaForBusiness18NameInUseExceptionV","text":"NameInUseException"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsalexaforbusiness\/updatenetworkprofileoutputerror\/nameinuseexception(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/UpdateNetworkProfileOutputError\/nameInUseException(_:)","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"nameInUseException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"NameInUseException","preciseIdentifier":"s:19AWSAlexaForBusiness18NameInUseExceptionV"},{"kind":"text","text":")"}],"title":"UpdateNetworkProfileOutputError.nameInUseException(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:19AWSAlexaForBusiness31UpdateNetworkProfileOutputErrorO18nameInUseExceptionyAcA04NamejkL0VcACmF","modules":[{"name":"AWSAlexaForBusiness"}]},"hierarchy":{"paths":[["doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness","doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/UpdateNetworkProfileOutputError"]]},"references":{"doc://AWSAlexaForBusiness/documentation/AWSAlexaForBusiness/NameInUseException":{"role":"symbol","title":"NameInUseException","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"NameInUseException"}],"abstract":[{"type":"text","text":"The name sent in the request is already in use."}],"identifier":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/NameInUseException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NameInUseException"}],"url":"\/documentation\/awsalexaforbusiness\/nameinuseexception"},"doc://AWSAlexaForBusiness/documentation/AWSAlexaForBusiness/UpdateNetworkProfileOutputError":{"role":"symbol","title":"UpdateNetworkProfileOutputError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"UpdateNetworkProfileOutputError"}],"abstract":[],"identifier":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/UpdateNetworkProfileOutputError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"UpdateNetworkProfileOutputError"}],"url":"\/documentation\/awsalexaforbusiness\/updatenetworkprofileoutputerror"},"doc://AWSAlexaForBusiness/documentation/AWSAlexaForBusiness":{"role":"collection","title":"AWSAlexaForBusiness","abstract":[],"identifier":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness","kind":"symbol","type":"topic","url":"\/documentation\/awsalexaforbusiness"},"doc://AWSAlexaForBusiness/documentation/AWSAlexaForBusiness/UpdateNetworkProfileOutputError/nameInUseException(_:)":{"role":"symbol","title":"UpdateNetworkProfileOutputError.nameInUseException(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"nameInUseException"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"NameInUseException","preciseIdentifier":"s:19AWSAlexaForBusiness18NameInUseExceptionV"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/AWSAlexaForBusiness\/documentation\/AWSAlexaForBusiness\/UpdateNetworkProfileOutputError\/nameInUseException(_:)","kind":"symbol","type":"topic","url":"\/documentation\/awsalexaforbusiness\/updatenetworkprofileoutputerror\/nameinuseexception(_:)"}}}